plugin_tests/client/annotationSpec.js

```diff
@@ -170,7 +169,91 @@ $(function () {
                 });
             });
         });
+    });
+
+    describe('Download view and region of interest', function () {
```

Comment:
Can you indent this entire describe block?  This will keep the outer describe block as a single block rather than divided into multiple describe blocks with the same description.

Reply:
Do you mean put this entire block into the discribe(Annotation tests) and remove the second discribe(Annotation tests) in order to keep only one block with this description ?

Reply:
Yes.  I'll just push the change.